src/pat/tooltip/tooltip.js

```diff
@@ -26,7 +26,7 @@ define([
     parser.addArgument("height", "auto", ["auto", "max"]);
     parser.addArgument("trigger", "click", ["click", "hover"]);
     parser.addArgument("closing", "auto", ["auto", "sticky", "close-button"]);
-    parser.addArgument("source", "title", ["auto", "ajax", "content", "title"]);
+    parser.addArgument("source", "title", ["auto", "ajax", "content_html", "title"]);
```

Comment:
@garbas You are changing content to content_html, doesn't that break any existing use of that argument? Should it be ["auto", "ajax", "content", "content-html", "title"] instead?